lang/en/product-types.php

```diff
@@ -5,4 +5,6 @@
 return [
     ProductType::EDIBLE->value => 'Edible',
     ProductType::GENERIC->value => 'Generic',
+    ProductType::PET_FOOD->value => 'Pet-food',
+    ProductType::PET_TOY->value => 'Pet-toy',
 ];
```

Comment:
Here the string will be what's displayed for this value, so you can just make them `Pet Food` and `Pet Toy`.